core/src/main/scala/org/broadinstitute/dsde/rawls/Boot.scala

```diff
@@ -133,6 +133,9 @@ object Boot extends App with LazyLogging {
       workbenchMetricBaseName = metricsPrefix
     )
 
+    //TODO: read this value from system properites after https://github.com/broadinstitute/firecloud-develop/pull/607/files goes in
```

Comment:
I merged 607. But make sure you give a default in case the system property is not defined.